clients/csrf/csrf_token_manager.go

```diff
@@ -0,0 +1,11 @@
+package csrf
+
+import "net/http"
+
+type CsrfTokenManager interface {
+	setCsrfToken() error
+	initializeToken(force bool, url string) error
```

Comment:
force method param does not mean anything for a person looking at the code for the first time.

Reply:
Renamed to -> forceInitializing